packages/p2p/src/components/page-return/page-return.jsx

```diff
@@ -7,7 +7,7 @@ import './page-return.scss';
 const PageReturn = ({ className, onClick, page_title }) => {
     return (
         <div className={classNames('page-return', className)} data-testid='order-details-wrapper-desktop'>
-            <div onClick={onClick} className='page-return__button'>
+            <div onClick={onClick} className='page-return__button' data-testid='dt_page_return_icon'>
                 <Icon icon='IcArrowLeftBold' size={16} />
             </div>
             <Text size='s' color='general' line_height='m' weight='bold'>
```

Comment:
```suggestion
            <Text color='general' weight='bold'>
```

Reply:
this was already done as part of page-return component refactoring.